phpcs.xml.dist

```diff
@@ -147,7 +147,7 @@
 	     provided as a comma-delimited list. -->
 	<rule ref="WordPress.WP.I18n">
 		<properties>
-			<property name="text_domain" type="array" value="wporg-learn,wporg_learn" />
+			<property name="text_domain" type="array" value="wporg-learn,wporg_learn,sensei-lms" />
```

Comment:
I didn't see any value in these strings in Sensei templates being translated a second time for our theme, but open to feedback